lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1904,8 +1904,10 @@ def _resolve_marketplace_image(
                             f"{marketplace.offer} {marketplace.sku} in {location}"
                         )
                     else:
+                        # use the same sort approach as Az CLI.
+                        versioned_images.sort(key=lambda x: parse(x.name), reverse=True)
                         # any one should be the same to get purchase plan
```

Comment:
Does this comment still add any useful information? This function is not being used for the sole purpose of getting the purchase plan information.

Reply:
Good catch